htdocs/adherents/note.php

```diff
@@ -68,15 +68,20 @@
 	$caneditfieldmember = $user->rights->adherent->creer;
 }
 
+$hookmanager->initHooks(array('membernote'));
+
 // Security check
 $result = restrictedArea($user, 'adherent', $object->id, '', '', 'socid', 'rowid', 0);
 
-
 /*
  * Actions
  */
+$reshook = $hookmanager->executeHooks('doActions', array(), $object, $action); // Note that $action and $object may have been modified by some hooks
+if ($reshook < 0)
```

Comment:
To match our PSR and checkstyle rules, can you replace with
if ($reshook < 0) {
...
}

The { must be mentionned, even if there is only one instruction into if.